build.sbt

```diff
@@ -2,110 +2,75 @@ import com.typesafe.sbt.packager.docker.Cmd
-  "org.typelevel"              %% "cats-core"                   % catsVersion,
-  "org.typelevel"              %% "cats-testkit"                % catsVersion,
-  "ch.qos.logback"             % "logback-classic"              % "1.2.3" % Runtime,
-  "net.logstash.logback"       % "logstash-logback-encoder"     % "6.1" % Runtime,
-  "org.codehaus.janino"        % "janino"                       % "3.0.13" % Runtime,
-  "com.github.pureconfig"      %% "pureconfig"                  % pureConfigVersion,
-  "com.github.pureconfig"      %% "pureconfig-cats"             % pureConfigVersion,
-  "io.kamon"                   %% "kamon-core"                  % kamonVersion,
-  "io.kamon"                   %% "kamon-akka"                  % kamonVersion,
-  "io.kamon"                   %% "kamon-prometheus"            % kamonVersion,
-  "eu.timepit"                 %% "refined"                     % refinedVersion,
-  "eu.timepit"                 %% "refined-pureconfig"          % refinedVersion,
-  "eu.timepit"                 %% "refined-scalacheck"          % refinedVersion,
-  "org.apache.kafka"           %% "kafka"                       % kafkaVersion % Test,
-  "org.scalatest"              %% "scalatest"                   % "3.0.8" % Test,
-  "com.typesafe.akka"          %% "akka-testkit"                % akkaVersion % Test,
-  "com.typesafe.akka"          %% "akka-stream-testkit"         % akkaVersion % Test,
-  "com.danielasfregola"        %% "random-data-generator"       % "2.7" % Test,
-  "com.47deg"                  %% "scalacheck-toolbox-datetime" % "0.2.5" % Test,
-  "org.mockito"                % "mockito-all"                  % "1.10.19" % Test,
-  "org.zalando"                %% "grafter"                     % "2.6.1" % Test,
-  "io.github.embeddedkafka"    %% "embedded-kafka"              % "2.6.0" % Test
-)
+Test / testOptions += Tests.Argument(TestFrameworks.ScalaTest, "-oF")
 
 val commonSettings = Seq(
-  organization := "com.sky",
-  scalaVersion := "2.12.10",
-  libraryDependencies += "com.sksamuel.avro4s" %% "avro4s-core" % "2.0.2"
+  organization                                 := "com.sky",
+  scalaVersion                                 := "2.13.8",
+  libraryDependencies += "com.sksamuel.avro4s" %% "avro4s-core" % "4.0.12"
 )
 
 lazy val dockerSettings = Seq(
-  packageName in Docker := "kafka-message-scheduler",
-  dockerBaseImage := "alpine:3.13.2",
-  dockerRepository := Some("skyuk"),
-  dockerLabels := Map("maintainer" -> "Sky"),
-  dockerUpdateLatest := true,
+  Docker / packageName := "kafka-message-scheduler",
+  dockerBaseImage      := "alpine:3.15.0",
+  dockerRepository     := Some("skyuk"),
+  dockerLabels         := Map("maintainer" -> "Sky"),
+  dockerUpdateLatest   := true,
   dockerCommands ++= Seq(
     Cmd("USER", "root"),
-    Cmd("RUN", "apk add --no-cache bash eudev openjdk11-jre")
+    Cmd("RUN", "apk add --no-cache bash eudev openjdk17-jre")
```

Comment:
Can probably remove `eudev` by disabling kamon host metrics, we don't use them anyway